index.js

```diff
@@ -27,40 +26,70 @@ module.exports = {
     }
 
     this.app = app;
- },
+  },
+
+  postprocessTree(type, tree) {
+    if (type !== 'all') {
+      return tree;
+    }
+
+    return this._prerenderTree(tree);
+  },
+
+  /**
+   * This function is *not* called by ember-cli directly, but supposed to be imported by an app to wrap the app's
+   * tree, to add the prerendered HTML files. This workaround is currently needed for Embroider-based builds that
+   * don't support the `postprocessTree('all', tree)` hook used here.
+   */
+  prerender(app, tree) {
+    let storybookAddon = app.project.addons.find(
+      ({ name }) => name === '@storybook/ember-cli-storybook'
+    );
+
+    if (!storybookAddon) {
+      throw new Error(
+        "Could not find initialized ember-cli-storybook addon. It must be part of your app's dependencies!"
+      );
+    }
```

Comment:
can't we just use `this`? 🤔

Reply:
I've taken this example from what @ef4 did on this MR https://github.com/ef4/prember/pull/64/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R22 

using `this` should work but I would rather stay as close as what he did since I'm still not 100% comfortable with Embroider yet and more specifically what it does under the hood and potential edge-case